ocs_ci/utility/prometheus.py

```diff
@@ -172,6 +173,68 @@ def get(self, resource, payload=None):
         )
         return response
 
+    def query_range(self, query, start, end, step, timeout=None, validate=True):
+        """
+        Perform Prometheus `range query`_. This is a simple wrapper over
+        ``get()`` method with plumbing code for range queries, additional
+        validation and logging.
+
+        Args:
+            query (str): Prometheus expression query string.
+            start (str): start timestamp (rfc3339 or unix timestamp)
+            end (str): end timestamp (rfc3339 or unix timestamp)
+            step (float): Query resolution step width as float number of
+                seconds.
+            timeout (str): Evaluation timeout in duration format. Optional.
+            validate (bool): Perform basic validation on the response.
+                Optional, ``True`` is the default. Use ``False`` when you
+                expect query to fail eg. during negative testing.
+
+        Returns:
+            list: result of the query
+
+        .. _`range query`: https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries
+        """
+        query_payload = {
+            'query': query,
+            'start': start,
+            'end': end,
+            'step': step}
+        if timeout is not None:
+            query_payload['timeout'] = timeout
+        # Human readable summary of the query (details are logged by get
+        # method itself with debug level).
+        logger.info((
```

Comment:
no need for double brackets here I guess

Reply:
Ah, good point. That said, I no longer have time to polish the code that much.